Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+/// Provides helper functions to test the SEPA Direct Debit flow in sandbox.
```

Comment:
Same comment as last PR regarding - do we want to call out that this isn't under version control?

Reply:
Added: https://github.com/braintree/braintree_ios/pull/913/commits/6fe0c36815e5aa7cc1ab531c0f0d57a650151664